### ISSUE TITLE
Fix unit tests in Csla.test

### ISSUE THREAD

**Issue by russblair** [MarimerLLC/csla#3395]
**Is your feature request related to a problem? Please describe.**
Csla unit tests in Csla.test not passing.

**Describe the solution you'd like**
Fix tests and or code.

**Describe alternatives you've considered**
None.

**Additional context**
Rocky, please assign this ticket to me.

**Comment by rockfordlhotka** [comment]
@russblair I invited you to my csla fork, because I've updated the `csla.tests` project to net8.0 and added it to the `csla.test` solution (which is run by the CI build).

My plan is to go through the failing tests one-by-one to either get them running or mark them as `Ignore` or delete them (if necessary).

I'll start from the top. If you have time this week and want to start from the bottom, that's excellent - hopefully we meet in the middle.

If you don't have time right before the Christmas/New Years holidays I fully understand!
